Samples/Server/Directx-Multithreaded/Deploy/deploy-rendering-server.ps1

```diff
@@ -1,6 +1,36 @@
-$gpuDriverVersion = "385.08" 
-$gpuDriverSuffix = "-tesla-desktop-winserver2016-international-whql.exe"
-$gpuDriverBaseUri = "http://us.download.nvidia.com/Windows/Quadro_Certified/"
+if($args.Length -gt 0)
```

Comment:
`-gt 1` now, right?

Reply:
I added this as a check for anyone who may use the deploy script directly, outside of VS.  We actually never used the arg we were passing previously :)

Reply:
makes sense! let's remove the check of update the condition, but then this looks great! 🔥 🚀

Reply:
Ahh, now I understand what your previous comment was.  Totally make sense, will update.